cmake/dependencies/common.cmake

```diff
@@ -30,3 +30,14 @@ target_sources(ImGui
 )
 
 target_include_directories(ImGui PUBLIC ${imgui_SOURCE_DIR} ${imgui_SOURCE_DIR}/backends PRIVATE ${SDL2_INCLUDE_DIRS})
+
+# ========= StormLib =============
+if(TRUE)
```

Comment:
this will be replaced with an actual condition in https://github.com/Kenix3/libultraship/issues/483